### ISSUE TITLE
[mediaqueries-4] order of prefixes

### ISSUE THREAD

**Issue by gsnedders** [w3c/csswg-drafts#7330]
https://drafts.csswg.org/mediaqueries-4/#mq-min-max defines min/max prefixes for range features.

https://www.w3.org/TR/CSS2/syndata.html#vendor-keywords reserves prefixes for vendor-specific extensions, at least for keywords and property names.

While admitting that media features aren't keywords or property names, the reality is that there are prefixed ones used in the wild (e.g., `-webkit-device-pixel-ratio`), and we should define somehow how prefixes get applied to `device-pixel-ratio`.

Should we have `min--webkit-device-pixel-ratio` or `-webkit-min-device-pixel-ratio`?

Clearly, the latter is the one that is implemented in every major browser engine, but there's nothing that actually says the prefixes should be applied in that order, and the Compat spec feels a need to make this explicit: https://compat.spec.whatwg.org/#css-media-queries-webkit-device-pixel-ratio

**Comment by tabatkins** [comment]
Yeah should definitely be the latter, both for consistency with implementations and just general coherence.

Hm, I suppose this is something that should be inlined into MQ.
